documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -864,6 +864,63 @@ The following example shows a typical transaction event message:
 }
 ----
 
+[[oracle-logminer-mining-strategies]]
+=== LogMiner Mining Strategies
+
+Oracle redo log entries do not contain the physical SQL performed by a user for DML-based changes.
+Instead, a redo entry holds a set of change vectors and a set of object identifiers that represent the tablespace, table, and columns related to these vectors.
+In other words, you won't find the names of schemas, tables, or columns in these redo entries.
+
+The {prodname} Oracle connector uses the xref:oracle-property-log-mining-strategy[`log.mining.strategy`] configuration property to control how Oracle LogMiner will handle the look-up of the object identifiers in the change vectors.
+Certain strategies are more reliable than others for specific corner cases around schema changes; however, there are performance and overhead implications that should be considered.
+
+==== Writing the data dictionary to redo logs
+
+The default strategy is called `redo_log_catalog`, and this works by flushing a copy of the data dictionary to the redo logs immediately upon each redo log switch.
+This is the most reliable strategy for tracking schema changes interwoven with data changes because Oracle LogMiner has a way to interpolate between the starting and ending data dictionary states across a series of change vectors.
+
+However, this mode is also the most experience because it requires several key steps to function.
```

Comment:
```suggestion
However, the `redo_log_catalog` mode is also the most expensive, because it requires several key steps to function.
```
Word choice?  Is this what you meant?